### DISCUSSION TITLE
Score is listed twice in lazer client.

### DISCUSSION THREAD

**Discussion by Inconsist** [ppy/osu#22247] [Q&A]
![image](https://user-images.githubusercontent.com/87947227/212778531-311dd122-7d06-4f5a-9ae9-12c1ab2ad5bf.png)

on the [website](https://lazer.ppy.sh/users/17637346) however its
![image](https://user-images.githubusercontent.com/87947227/212777915-18c52ec7-1c74-4e81-95ea-e81cdcf618a0.png)

and after a restart of the client the scores were in alignement with the website again?
I looked for a similar bug but didnt find anything. Is this grave enough to be worth its own issue or just something that can happen since a simple restart fixed it?

**Comment by bdach** [comment]
Did you perhaps click the "Load More" button when this happened?

I'm thinking this may be a transient pagination failure on the osu-web side (score used to be on page 1, then was indexed in elastic, and thus was downgraded to page 2 before page 2 was requested).

**Reply by Inconsist** [reply]
This screenshot was made with the list fully extended but I cant remember if it was right on the edge.

But wouldnt that reproduce even after I restarted the game?

**Reply by bdach** [reply]
It wouldn't, since after the index updated, it would be in a fully consistent state again. It wouldn't appear on page 1, but be on page 2 again, only once.

**Reply by Inconsist** [reply]
Yup that could explain it.
